### ISSUE TITLE
Regexp fails on Fiscal Code with Omocodia

### ISSUE THREAD

**Issue by kijubi95** [robertogallea/laravel-codicefiscale#23]
Hello.
I have a fiscal code with omocodia, precisely L4L7 instead of L407 (Treviso). 
It gives me an exception during the check **CheckForWrongCode** when using the regular expression in the page.

The fiscal code is: XXXXXX##X##L4L7T ( X for letters, # for numbers). 

The regexp fails in the bold part: 
/^[a-z]{6}[0-9]{2}[a-z][0-9]{2}[a-z]**[0-9]{2}**[0-9a-z][a-z]$/i

A temporary solution I am using is:
^[a-z]{6}[0-9]{2}[a-z][0-9]{2}[a-z]**[0-9a-z]{3}**[a-z]$/i

I think that all the indexes where we can find a char replaced with an omocodia code sould be replaced
in the regexp from [0-9] to [0-9a-z]. 

Like: ^[a-z]{6}[0-9a-z]{2}[a-z][0-9a-z]{2}[a-z][0-9a-z]{3}[a-z]$/i

**Comment by robertogallea** [comment]
Hi,
will you be available for submitting a PR with a test reproducing your issue?

Thank you!

**Comment by kijubi95** [comment]
I am sorry, I am not in an enviroment where I can fork the project and make a pull request. 
Also, I cannot write the fiscal code in my case for privacy reason, but I'll use a fake one.

I'll copy the code here. 
This one is correct:
```
$validator = new CodiceFiscale(); 
$validator->parse("MKJRLA80A01L407X")
```
This one fails:
```
$validator = new CodiceFiscale(); 
$validator->parse("MKJRLA80A01L4L7I")
```

**Comment by robertogallea** [comment]
Fixed in v1.7.1. Please confirm.

Thanks!

**Comment by aureliomakaj** [comment]
Thank you very much!
Everything works fine now.
